packages/roosterjs-editor-plugins/lib/plugins/Paste/Paste.ts

```diff
@@ -126,3 +131,10 @@ function isWordDocument(htmlAttributes: Record<string, string>) {
         htmlAttributes[PROG_ID_NAME] == WORD_PROG_ID
     );
 }
+
+function getWordOnlineTable(fragment: DocumentFragment) {
```

Comment:
I think we should check for all the tables, not just one.

Reply:
agree.